generated/nidaqmx/_library_interpreter.py

```diff
@@ -106,6 +108,9 @@ def auto_configure_cdaq_sync_connections(
     def calculate_reverse_poly_coeff(
             self, forward_coeffs, num_forward_coeffs_in, min_val_x, max_val_x,
             num_points_to_compute, reverse_poly_order):
+        size = len(forward_coeffs) if (reverse_poly_order < 0) else reverse_poly_order + 1
```

Comment:
If we are adding a special case for this function, then we should also add some test coverage.

It's an easy function to test because it doesn't require any hardware or tasks. It just does some math and returns the result.

Reply:
Thanks for the suggestion, i have added the tests